templates/security/esm.html

```diff
@@ -8,318 +8,560 @@
-        <div class="p-heading-icon--small">
-          <div class="p-heading-icon__header">
-            {{
-              image(
-              url="https://assets.ubuntu.com/v1/fc781c43-Case+study+-+white.svg",
-              alt="",
-              height="32",
-              width="32",
-              hi_def=True,
-              loading="lazy",
-              attrs={"class": "p-heading-icon__img"},
-              ) | safe
-            }}
-            <p class="p-heading--4">Case study</p>
-          </div>
-          <h4>
-            TIM maintains system security and client confidence with ESM
-          </h4>
-          <p>
-            To ensure the ongoing security of Ubuntu 14.04 LTS machines, ESM provided TIM, the world's largest trade recommendations network, the freedom to upgrade within their own timeframe. This approach saved time and money for this finserv organisation.
-          </p>
-          <p><a href="/engage/case-study-acuris" class="p-button">Read more in the case study</a></p>
-        </div>
+<section class="p-strip is-deep u-no-padding--top">
+    <div class="u-fixed-width">
+      <hr />
+      <h2>What's the difference?</h2>
+      <div class="esm-table p-strip is-shallow">
+        <table class="p-table">
+          <thead>
+            <tr>
+              <th width="51%">Security patching<br /> <span style="text-transform: initial; color: #666666;">(Coverage for critical, high and selected medium CVEs)</span></th>
+              <th width="15%" class="u-align--left">Ubuntu LTS</th>
+              <th width="24%" class="u-align--left">Ubuntu Pro (Infra-only)<br /> <span style="text-transform: initial; color: #666666;">(Previously known as ”Ubuntu Advantage for Infrastructure”)</span></th>
+              <th width="10%" class="u-align--left">Ubuntu Pro</th>
+            </tr>
+          </thead>
+          <tbody>
+            <tr>
+              <td>Over 2,300 packages in Ubuntu Main repository</td>
+              <td class="u-align--left">5 years </td>
+              <td class="u-align--left">10 years</td>
+              <td class="u-align--left">10 years</td>
+            </tr>
+            <tr>
+              <td>Over 23,000 packages in Ubuntu Universe repository</td>
+              <td class="u-align--left">Best effort</td>
+              <td class="u-align--left">Best effort</td>
+              <td class="u-align--left">10 years</td>
+            </tr>
+          </tbody>
+        </table>
       </div>
     </div>
+
+    <div class="u-fixed-width">
+      <table aria-hidden="true" class="p-table--esm-chart">
+        <thead class="u-hidden">
+          <th width="6%"></th>
+          <th colspan="2" width="25%">&nbsp;</th>
+          <th colspan="10" width="65%">&nbsp;</th>
+        </thead>
+        <tbody>
+          <tr height="254px">
+            <td class="u-align--right u-table-border--top u-no-padding-right" style="padding-top: 0"><strong>2,300</strong><br /><strong>23,000</strong></td>
```

Comment:
@carkod  are you confident this will make sense in a screen reader? Why separate number and description, instead of having them in two separate rows?

Reply:
Because that's how it aligns to the design. The screen reader will not read this, just like it won't read a D3.js graph

Reply:
sorry my question is, why not have two trs, first td - the number, second td - the text? And what do you mean a screen reader won't read it - is it flagged to be ignored by a screen reader?